configure.py

```diff
@@ -1463,9 +1472,9 @@ def write_config(data, name):
       icu_ver_major = m.group(1)
   if not icu_ver_major:
     error('Could not read U_ICU_VERSION_SHORT version from %s' % uvernum_h)
-  elif int(icu_ver_major) < icu_versions["minimum_icu"]:
-    error('icu4c v%d.x is too old, v%d.x or later is required.' % (int(icu_ver_major),
-                                                                  icu_versions["minimum_icu"]))
+  elif int(icu_ver_major) < icu_versions['minimum_icu']:
+    error('icu4c v%d.x is too old, v%d.x or later is required.' %
```

Comment:
`%s` so you don't have to coerce to int? That's what the code on line 1355 does.

Reply:
Addressed